common/exf.c

```diff
@@ -199,7 +199,14 @@ file_init(SCR *sp, FREF *frp, char *rcv_name, int flags)
 		if (!LF_ISSET(FS_OPENERR))
 			F_SET(frp, FR_NEWFILE);
 
+#if defined HAVE_STRUCT_STAT_ST_MTIMESPEC
+		ep->mtim = sb.st_mtimespec;
+#elif defined HAVE_STRUCT_STAT_ST_MTIM
 		ep->mtim = sb.st_mtim;
+#else
+		ep->mtim.tv_sec = sb.st_mtime;
```

Comment:
Do we care?

Reply:
I could have simply put `#error` here but I thought it wasn't too much effort to implement this case.